hive-llap/start_llap.sh

```diff
@@ -42,13 +42,12 @@ function start_llap(){
 
         echo "Setting Parameters for LLAP start"
         
-        LLAP_SIZE=$NODE_MANAGER_MEMORY
+        local LLAP_SIZE=$NODE_MANAGER_MEMORY
```

Comment:
local variables should be in lower case. `LLAP_SIZE` -> `llap_size`

Reply:
changed